SqlInstall.ps1

```diff
@@ -0,0 +1,433 @@
+                Write-Message -Level Verbose -Message "DataVolume moved to " $DataVolume
+            }
+            
+            Write-Message -Level Verbose -Message "logvolume: " $LogVolume
+            $NewLogVolume = Read-Host "Your logvolume: "
+            If ([string]::IsNullOrEmpty($NewLogVolume)) {
+                Write-Message -Level Verbose -Message "Logvolume remains on " $LogVolume
+            }
+            else {
+                $NewLogVolume = $NewLogVolume -replace "\\$"
+                $NewLogVolume = $NewLogVolume -replace ":$"
+                $LogVolume = $NewLogVolume
+                Write-Message -Level Verbose -Message "LogVolume moved to " $LogVolume
+            }
+
+
+            Write-Message -Level Verbose -Message "TempVolume: " $TempVolume
+            $NewTempVolume = Read-Host "Your TempVolume: "
+            If ([string]::IsNullOrEmpty($NewTempVolume)) {
+                Write-Message -Level Verbose -Message "TempVolume remains on " $TempVolume
+            }
+            else {
+                $NewTempVolume = $NewTempVolume -replace "\\$"
+                $NewTempVolume = $NewTempVolume -replace ":$"
+                $TempVolume = $NewTempVolume
+                Write-Message -Level Verbose -Message "TempVolume moved to " $TempVolume
+            }
+
+            Write-Message -Level Verbose -Message "AppVolume: " $AppVolume
+            $NewAppVolume = Read-Host "Your AppVolume: "
+            If ([string]::IsNullOrEmpty($NewAppVolume)) {
+                Write-Message -Level Verbose -Message "AppVolume remains on " $AppVolume
+            }
+            else {
+                $NewAppVolume = $NewAppVolume -replace "\\$"
+                $NewAppVolume = $NewAppVolume -replace ":$"
+                $AppVolume = $NewAppVolume
+                Write-Message -Level Verbose -Message "AppVolume moved to " $AppVolume
+            }
+
+            Write-Message -Level Verbose -Message "BackupVolume: " $BackupVolume
+            $NewBackupVolume = Read-Host "Your BackupVolume: "
+            If ([string]::IsNullOrEmpty($NewBackupVolume)) {
+                Write-Message -Level Verbose -Message "BackupVolume remains on " $BackupVolume
+            }
+            else {
+                $NewBackupVolume = $NewBackupVolume -replace "\\$"
+                $NewBackupVolume = $NewBackupVolume -replace ":$"
+                $BackupVolume = $NewBackupVolume
+                Write-Message -Level Verbose -Message "BackupVolume moved to " $BackupVolume
+            }
+        }
+        Default {Write-Message -Level Verbose -Message "Drives agreed, continuing"; }
+    }
+
+
+    # Out-File -FilePath C:\Temp\ConfigurationFile2.ini -InputObject $startScript
+
+    (Get-Content -Path $configini).Replace('SQLBACKUPDIR="E:\Program Files\Microsoft SQL Server\MSSQL12.AXIANSDB01\MSSQL\Backup"', 'SQLBACKUPDIR="' + $BackupVolume + ':\Program Files\Microsoft SQL Server\MSSQL12.AXIANSDB01\MSSQL\Backup"') | Out-File $configini
```

Comment:
It looks like this will overwrite the config file in the module directory. Please use a temporary directory instead.

````
$temp = ([System.IO.Path]::GetTempPath()).TrimEnd("\")
$tempini = "$temp\config.ini"
````

Reply:
It shouldn't overwrite it, earlier in the code, the chosen ini file gets copied to a different location (around line 127 this process takes place :))